relay/kusama/src/lib.rs

```diff
@@ -185,10 +185,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: alloc::borrow::Cow::Borrowed("kusama"),
 	impl_name: alloc::borrow::Cow::Borrowed("parity-kusama"),
 	authoring_version: 2,
-	spec_version: 1_005_001,
+	spec_version: 1_006_000,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
-	transaction_version: 26,
+	transaction_version: 27,
```

Comment:
Since it was confirmed that we no longer use/need `transaction_version`, I’d suggest reverting it here everywhere - just to be safe and avoid any unexpected surprises - maybe too paranoid :)